packages/vue-app/template/mixins/fetch.client.js

```diff
@@ -38,7 +38,7 @@ function created() {
 
   // Hydrate component
   this._hydrated = true
-  this._fetchKey = +this.$vnode.elm.dataset.fetchKey
+  this._fetchKey = this.$vnode.elm.dataset.fetchKey
```

Comment:
What happens in case of hydration mismatch? (key not available). What if throwing a clear error about this?

Reply:
I think we are guaranteed for key to exist: https://github.com/nuxt/nuxt.js/blob/c4bc0c9cd83e483954d7c218c19a236c40c4718a/packages%2Fvue-app%2Ftemplate%2Fmixins%2Ffetch.client.js#L4